OloEngine/src/OloEngine/Physics3D/JoltShapes.cpp

```
@@ -690,52 +579,100 @@ namespace OloEngine
 
     // Private helper implementations
 
-    f32 JoltShapes::ComputeMeshVolume(AssetHandle colliderAsset, const glm::vec3& scale)
+    MeshMassProperties JoltShapes::ComputeTriangleMeshMassProperties(std::span<const Vertex> vertices,
```

Cppcheck (2.20.0)

[style] The function 'ComputeTriangleMeshMassProperties' is never used.

(unusedFunction)